tests/sentry/models/test_notificationsetting.py

```diff
@@ -19,15 +22,18 @@ def _get_kwargs(kwargs):
     )
 
 
-def assert_no_notification_settings(**kwargs):
-    assert NotificationSetting.objects._filter(**kwargs).count() == 0
+def assert_no_notification_settings():
+    assert NotificationSetting.objects.all().count() == 0
+    assert NotificationSettingOption.objects.all().count() == 0
+    assert NotificationSettingProvider.objects.all().count() == 0
 
 
 def create_setting(**kwargs):
```

Comment:
I _think_ we prefer using `@with_feature("organizations:notifications-double-write")` instead - it more closely mocks feature flag usage in prod

Reply:
@snigdhas what's the difference? I'm doing it here because I only need it for `update_settings` to populate the tables via the double write